proxies/shared/resources/jsc/SetResponseDefaults.js

```diff
@@ -17,3 +17,15 @@ context.setVariable("error.header.Cache-Control", "no-cache, no-store, must-reva
 // set our x-content-type-options-header
 context.setVariable("response.header.X-Content-Type-Options", "nosniff");
 context.setVariable("error.header.X-Content-Type-Options", "nosniff");
+
+// format errors object
+try {
+  const content = context.getVariable("error.content")
+  const errors = JSON.parse(content).errors
```

Comment:
Could we not just `context.setVariable("error.context", JSON.stringify(JSON.parse(content)))`?

Reply:
Also what about valid content from the message batches endpoint, see `AssignMessage.MessageBatches.Create.Response.xml`?